### PR TITLE
Update ways_to_contribute.rst

### PR DESCRIPTION
**Pull request by chaytanyasinha**
Update ways_to_contribute.rst by solving small grammatical errors.

### Summary
Update ways_to_contribute.rst by solving small grammatical errors.

…

### Reviewer guidance

…

### References

…

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by indirectlylit — COMMENTED** [review]
thanks for the contribution!

This isn't documented, but we actually try to roughly follow the [Google Material writing guidelines](https://material.io/design/communication/writing.html), which by convention avoid using periods in bulleted lists:

> image

This isn't a hard-and-fast rule but if anything on this page, I'd actually lean towards removing the period on line 13 rather than adding it to the other bullets

**Comment by radinamatic** [comment]
Agreed with @indirectlylit, do not add any periods, just delete the one at the end of the first item of the _Talk to us_ bullet list.

**Comment by chaytanyasinha** [comment]
@indirectlylit @radinamatic Thank you for the suggestions.

File: docs/contributing/ways_to_contribute.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ Talk to us
 
 * Get support in our `Community Forums <http://community.learningequality.org/>`__.
 * Email us at info@learningequality.org
```
Comment (jonboiser) [inline docs/contributing/ways_to_contribute.rst:14]:
```suggestion
* Email us at info@learningequality.org.
```

Comment (indirectlylit) [reply docs/contributing/ways_to_contribute.rst:14]: @jonboiser see above: https://github.com/learningequality/kolibri/pull/6992#pullrequestreview-425643165

**Comment by indirectlylit** [comment]
@chaytanyasinha I'm going to close this to clean up our outstanding PRs. Your contribution is appreciated though, thank you!

**Pull request closed without merging**